i18n/vi.yml

```diff
@@ -164,7 +164,6 @@ components:
   PlaceEditor:
     genericLocationPlaceholder: Tìm kiếm vị trí
     locationPlaceholder: Tìm kiếm vị trí {placeName}
-    namePlaceholder: Đặt tên địa điểm
```

Comment:
Does anyone know why this is going to be deleted?

Reply:
This string was deleted from the English file in #782.